.github/workflows/hosted_runners.yml

```diff
@@ -1244,3 +1244,16 @@ jobs:
       shell: powershell
       run: |
         rm -r -Force ${{ steps.build_paths.outputs.BINARY }}
+
+  # This job is here as a github status check -- it allows us to move the dependency from
+  # being on all the jobs to this single one.
+  mergeable:
```

Comment:
Can we update the name of this job to `process-mergeable-check`? `mergeable` sounds a tad too ambiguous.